ibm_catalog.json

```
@@ -43,6 +43,10 @@
         {
           "description": "Provisions and manages SSH keys for the VSIs so that you can securely administer the instances.\n",
           "title": "Provisions SSH keys"
+        },
+        {
+          "description": "Configures CBR to open traffic flow only from landing zone VPCs to specific cloud services for an existing CBR rule.\n",
```

```suggestion
          "description": "Configures context-based restriction rules to allow traffic to flow only from the landing zone VPCs to specific cloud services.\n",
```

I don't know what configuring CBR... for an existing rule means, so rewrote.